app/helpers/application_helper.rb

```diff
@@ -373,23 +373,27 @@ def update_otml_url_for(component, escape=true)
     end
   end
 
-  def report_link_for(reportable, action='report', link_text='Report ', title=nil)
-    return "" if reportable.respond_to?('reportable?') && !reportable.reportable?
+  def learner_report_link_for(learner, action='report', link_text='Report ', title=nil)
+    return "" if learner.respond_to?('reportable?') && !learner.reportable?
```

Comment:
If learner is always a portal_learner then it should always respond_to reportable.  So I think the first part of this condition can be removed (unless you found otherwise).